airgun/entities/contentviewfilter.py

```diff
@@ -184,6 +184,24 @@ def search_package_rule(self, cv_name, filter_name, query):
         )
         return view.content_tabs.rpms.search(query)
 
+    def remove_package_rule(self, cv_name, filter_name, rpm_name):
+        """Remove specific package rule from RPM content view filter.
+
+        :param str cv_name: content view name
+        :param str filter_name: content view filter name
+        :param str rpm_name: name of package used in rule to be removed
+        """
+        view = self.navigate_to(
+            self, 'Edit',
+            cv_name=cv_name,
+            filter_name=filter_name,
+        )
+        view.content_tabs.rpms.search(rpm_name)
+        view.content_tabs.rpms.table.row()[0].widget.fill(True)
```

Comment:
Why index instead of filtering?
Also, if i remember correctly, there might be multiple rules for 1 rpm name, so filtering by multiple columns (or at least support of custom queries) is preferable.

Reply:
Of course, I know about that =) but `view.content_tabs.rpms.table.row(rpm_name=rpm_name)` or `view.content_tabs.rpms.table.row(rpm_name__consists...)` doesn't work for me. There is possibility to go way you used in `update`, but I feel it is too much for that method, just too complex. I selected easiest way that will work in 95% of cases. Any idea?

Reply:
maybe moving filtering part of update outside in some private method and reusing in both helpers then? I'd rather like to see good method which will work with all future tests, especially as this implementation may hide errors (delete wrong filter successfully) and basically there's implementation of filtering ready.

Reply:
okay, I will try that idea
and index was used with the same idea as you use `ACTIONS_COLUMN` =), but yeah let's me test approach